apps/swirl-docs/src/components/Layout/CategoryNav.tsx

```diff
@@ -25,18 +31,84 @@ export function CategoryNav() {
   //   </ul>
   // );
 
-  const SubElement = ({ navItem }: { navItem: NavItem }) => (
-    <li
-      className={classNames("font-sm mb-4", "hover:text-border-info", {
-        "text-border-info": activePath === navItem.url,
-        "text-text-default": activePath !== navItem.url,
-      })}
-    >
-      <Link href={`${navItem.url}`}>
-        <a className="text-sm capitalize">{navItem.title}</a>
-      </Link>
-    </li>
-  );
+  const SubElement = ({ navItem }: { navItem: NavItem }) => {
+    const [isExpanded, setIsExpanded] = useState(false);
+
+    useEffect(() => {
+      if (activePath.includes(navItem.url)) {
+        setIsExpanded(true);
+      }
+    }, [navItem.url]);
+
+    return (
+      <li className={classNames("mb-2 py-2", { "max-h-10": !isExpanded })}>
```

Comment:
**issue:** See https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/ for collapsible navigation pattern. (aria-expanded, aria-controls).